cliquet/storage/postgresql/client.py

```diff
@@ -62,25 +69,38 @@ def create_from_config(config, prefix=''):
                    "Refer to installation section in documentation.")
         raise ImportWarning(message)
 
-    settings = config.get_settings()
-    url = settings[prefix + 'url']
+    from zope.sqlalchemy import ZopeTransactionExtension, invalidate
```

Comment:
no it's in `POSTGRESQL_REQUIRES`  (`requirements.txt` is more like a good old *known good set*)